analytics_data_api/v0/views/learners.py

```diff
@@ -124,16 +121,24 @@ class LearnerListView(LastUpdateMixin, CourseViewMixin, generics.ListAPIView):
     **Response Values**
 
         Returns a paginated list of learner metadata and engagement data.
-        Pagination data is returned in the top level of the returned JSON
-        object.
+
+        This request will return JSON or CSV data, depending on the request Accept header.
+        Default is JSON data.
+
+        If the returned object is JSON, pagination data is returned in the top level:
 
             * count: The number of learners that match the query.
             * page: The current one-indexed page number.
             * next: A hyperlink to the next page if one exists, otherwise null.
             * previous: A hyperlink to the previous page if one exists,
               otherwise null.
 
-        The 'results' key in the returned object maps to an array of
+        Pagination links are also returned in the response's header, e.g.
+            Link: <next_url>; rel="next", <previous_url>; rel="prev";
+
+        If the returned object is CSV, only the 'results' are returned.
```

Comment:
Could we restructure this slightly? It's a little confusing to have "if the returned object is CSV, only the 'results' are returned" directly after the discussion of the pagination headers; it could be interpreted as saying that CSV results don't get pagination. Possibly structure like so:

- Talk about pagination headers
- Talk about the result, the contents of each list item, and how it's returned directly as a list when in CSV format
- Talk about how the result list is embedded in an object with extra pagination controls when JSON is used

Up to you, though.